scripts/deploy_single.sh

```
@@ -0,0 +1,154 @@
+#!/bin/bash
+set -e
+
+# Load environment variables
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+source "$PROJECT_ROOT/.env.production"
```

Shellcheck (0.10.0)

[info] Not following: ./.env.production was not specified as input (see shellcheck -x).

(SC1091)